tools/isobuild/meteor-npm.js

```diff
@@ -1053,20 +1053,6 @@ var getShrinkwrappedDependencies = function (dir) {
   return treeToDependencies(getShrinkwrappedDependenciesTree(dir));
 };
 
-const moduleDoesResolve = meteorNpm.moduleDoesResolve = (dep) => {
```

Comment:
This function was used only in `dev-bundle-helpers.js`, so I decided to remove it.